doc/fuzzing.md

```diff
@@ -146,23 +145,43 @@ Every single pull request submitted against the Bitcoin Core repo is automatical
 ## macOS hints for libFuzzer
 
 The default Clang/LLVM version supplied by Apple on macOS does not include
-fuzzing libraries, so macOS users will need to install a full version, for
-example using `brew install llvm`.
```

Comment:
Why remove the `brew` hint here? In other places the examples still include `brew` references.

Reply:
we're repeating the same a few lines below

Reply:
Do you mean the `lld` instruction? I find that confusing to keep that only there. I would prefer if you could keep the instruction here as well and maybe adapt the `lld` part to only talk about installing `lld` potentially also making clear that using brew is just an example, just like here.

Reply:
> Do you mean the lld instruction?

No, I mean we already have
```bash
brew install llvm lld
```
a few lines below, I don't see the point of announcing the same in the doc a few paragraphs before, too.

> keep the instruction here as well and maybe adapt the lld part 

What would be the advantage of sprinkling all the dependencies throughout multiple separate brew commands?